Angelina menu reviews table

ahjsuy opened the pull request:
Added the Table Component as well as its stories and tests

https://ucsb-cs156-w24.github.io/team03-w24-4pm-2/prs/66/storybook/?path=/story/components-menuitemreview-menuitemreviewform--create

![image](https://github.com/ucsb-cs156-w24/team03-w24-4pm-2/assets/92120716/a1ce6488-9e98-4bcb-9e4e-56d36d4eff4e)
![image](https://github.com/ucsb-cs156-w24/team03-w24-4pm-2/assets/92120716/b0227943-8aea-42b2-99a9-89f2d0d37949)
![image](https://github.com/ucsb-cs156-w24/team03-w24-4pm-2/assets/92120716/9982e78b-785c-4856-84eb-bf51390e5e3f)

Closes #38

rohankumar2025 reviewed (commented):
Not sure if we are supposed to make our own util file, but your MenuItemReviewTable.js file is using the utils from UCSBDateUtils which would call the api endpoint for ucsbdates?

ahjsuy commented:
> Not sure if we are supposed to make our own util file, but your MenuItemReviewTable.js file is using the utils from UCSBDateUtils which would call the api endpoint for ucsbdates?

I added its own util file and changed it in later prs

KylebKumar reviewed (approved):
LGTM
